renku/cli/dataset.py

```diff
@@ -104,6 +104,26 @@
         new-dir/
           new-filename
 
+Updating a dataset:
+
+After adding files from a remote Git repository, you can check for updates in
+those files by using ``renku dataset update`` command. This command checks all
+remote files and copies over new content if there are any. It also deletes
+files from the local dataset if they are deleted from the remote Git
```

Comment:
Hmm I hadn't thought of that before but I guess this will lead to some unexpected behavior. Is there an issue already for updating to a specific commit?

Reply:
Not yet. It was supposed to be part of this one but I decided to do it later. I'll create on when we close this one. I don't see though how that would help; if a file is deleted from the latest update eventually we need a way to deal with it.

Reply:
Yes sure, but right now I can imagine someone could do an update that deleted a file with no way to get back other than doing a git reset. If they were able to go to a previous commit it would be a non-issue.

Reply:
anyway, this is not the right place for this discussion I realize :)  We should add support for updating to a specific commit reasonably soon to avoid having users that get stuck in some strange state.

Reply:
As we discussed, I'll ask the user for deletion.